src/functions/utils.php

```diff
@@ -486,3 +486,34 @@ function tribe_post_excerpt( $post ) {
 		return wpautop( $excerpt );
 	}
 }
+
+/**
+ * Create a function to mock the real function if the extension or Beta is not present.
+ *
+ * @todo remove this function after WP 5.0 is released
+ *
+ */
+if ( ! function_exists( 'has_blocks' ) ) {
```

Comment:
Perhaps we can add the WP version compare to the conditional? When 5.0 is out we'll still have people using WP < 5.0 and they will still get a fatal if we remove it

Reply:
Oh that's right since even if 5.0 is there doesn't mean people is using.

However I think adding the conditional here for that version does not make too much difference as the main point is to make sure the function is there as even with the `gutenberg` extension this function can be present and adding an `||` conditional to the above does not provide more value IMHO, but let me know if you think otherwise.

Reply:
However we might need to update the wording around the `@todo`?

Reply:
Yeah, adding to the conditional won't do much. What I meant is basically, is that if we're mocking this function to prevent a fatal we cannot remove the function once 5.0 is out. Perhaps  just removing the `todo` is fine. Ideally people will update WP, but if they don't and they update our plugins they'll end up facing the fatal.